pages/index.js

```diff
@@ -25,12 +27,24 @@ const StyledWrapper = styled.div`
   justify-content: space-between;
 `;
 
+const StyledButtonWrapper = styled.div`
+  display: ${({ $show }) => ($show ? "flex" : "none")};
+  width: inherit;
+  justify-content: center;
+`;
+
 const StyledSpan = styled.span`
   font-size: 0.8rem;
 `;
 
 const StyledButton = styled.button`
-  width: 3rem;
+  display: ${({ $show }) => ($show ? "none" : "block")};
```

Comment:
The button hides itself when `$show` is `true`.  It's tying my brain in knots right now.